packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -404,6 +407,15 @@ class SnackBar extends StatefulWidget {
   /// Defaults to [Clip.hardEdge], and must not be null.
   final Clip clipBehavior;
 
+  /// The overflow threshold for action widgets.
```

Comment:
It's best to not just reiterate the property name here. That said, explaining what this property means in one pithy sentence is difficult. Seems like it would be something like:

If the width of the snackbar's [content] is greater than this percentage of the width of the snackbar less the width of its [action], then the [action] will appear below the [content].

And then below you'd have to explain what this means in terms of a few cases, what the "width of the action" actually means, likewise for the snackbar's width. I suppose it would also be worth mentioning at the outset that a layout invariant for the snackbar is that the content and the action do not overlap horizontally.

Reply:
Thanks - have reworded. this.